content/2.0/index.html

```diff
@@ -0,0 +1,117 @@
+---
+:layout: post
```

Comment:
FWIW, this is the wrong layout for anything that isn't a blog post. Generally `simplepage` is going to be what you'll want in the future.